test/compose/cdi_device/teardown.sh

```diff
@@ -0,0 +1,4 @@
+if ! is_rootless; then
+    umount /etc/cdi
+    rmdir /etc/cdi || true # do not return an error if the directory is non-empty
```

Comment:
if the directory existed before we delete it now which seems unexpected, I know it is not deleted if not empty but still.
I think just keeping the directory is better,

Reply:
dropped the rmdir